frontend/src/Main.purs

```diff
@@ -36,18 +36,22 @@ socketConfig appConfig = do
     -- socket
     actionChannel <- channel $ Ex.SocketConnected false
     let socketSignal = subscribe actionChannel :: Signal Ex.Action
+        pingSignal = every (10.0 * second) ~> const Ex.SocketPing
```

Comment:
Nice. 
I'm not sure how much is the server going to be handling a large number of connections + `SocketPing` every 10 second - just a thing to have in mind.